### PR TITLE
i18n(ko-KR): add `markdown-content-schema-validation-error.mdx`

### PR DESCRIPTION
**Pull request by jsparkdev**
#### Description (required)

-  add `markdown-content-schema-validation-error.mdx`

#### Related issues & labels (optional)

- Closes #
- Suggested label:

### CODE REVIEW

**Review by VoxelMC — APPROVED** [review]
Thank you for updating!
`As a non-native speaker, I reviewed this PR using the Docs Translation Review Checklist and it LGTM!`

I am not too familiar with the actual Korean translation guidelines, but I remember the word `frontmatter` being written in plain English before in some other translations. I could be wrong, however!

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks 🚀
